src/services/notifier/hooks/subscriptions.hook.ts

```diff
@@ -46,29 +46,25 @@ export default {
           ]
```

Comment:
we can remove the type type HashesByProviderAndConsumer = Record<string, Record<string, string[]>> which is not used anywhere